subgraph/auxiliary/subgraph.yaml

```diff
@@ -21,3 +21,32 @@ dataSources:
         - event: StateRootAvailable(uint256,bytes32)
           handler: handleStateRootAvailable
       file: ./src/AnchorMapping.ts
+
+  - kind: ethereum/contract
+    name: ERC20Cogateway
+    network: dev
+    source:
+      address: "0xdbbb5c822ccfac3ea49e32215565c6aafb257a30"
```

Comment:
Update in ticket that contract address needs to be updated after deployment and tag @deepesh-kn

Reply:
Commented on the ticket.